kratos/processes/levelset_convection_process.h

```diff
@@ -58,7 +63,7 @@ namespace Kratos
  * on the top of it
 */
 template< unsigned int TDim, class TSparseSpace, class TDenseSpace, class TLinearSolver >
-class LevelSetConvectionProcess
+class KRATOS_API(KRATOS_CORE) LevelSetConvectionProcess
```

Comment:
```suggestion
class LevelSetConvectionProcess
```
this doesn't work for header only classes